api/nakadi-event-bus-api.yaml

```diff
@@ -484,3 +851,123 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Event'
+
+  EventType:
+    description: An event type defines the schema and its runtime properties.
+    properties:
+      name:
+        type: string
+        description: |
+          Name of this EventType. The name for the EventType must follow the pattern
+          'organization:team:application:event-type-name'
+      topic:
+        type: string
+        description: |
+          References the name of the low-level topic that is holding the stream of Events for this EventType.
+          Usually is the same as the name property.
+      validation-strategies:
+        description: |
+          Determines the validation that has to be executed upon reception of Events of this type. Events are
+          rejected if any of the rules fail (see details of Problem response on the Event publishing methods).
+
+          Rule evaluation order is the same as in this array.
+        type: array
+        items:
+          $ref: '#/definitions/EventValidationStrategy'
+
+      enrichment-strategies:
+        description: |
+          Determines the enrichment to be performed on an Event upon reception. Enrichment is performed once
+          upon reception (and after validation) of an Event and is only possible on fields that are not
+          defined on the incoming Event.
+
+          See documentation for the write operation for details on behaviour in case of unsuccessful
+          enrichment.
+        type: array
+        items:
+          $ref: '#/definitions/EventEnrichmentStrategy'
+
+      partition-resolution-strategy:
+        description: |
+          Determines how the assignment of the event to a Partition should be handled.
+
+        type: object
+        $ref: '#/definitions/PartitionResolutionStrategy'
+
+      schema:
+        type: object
+        $ref: '#/definitions/EventTypeSchema'
+        description: |
+          The schema for this EventType. This is expected to be a json schema in yaml format (other formats
+          might be added in the future).
```

Comment:
What is the purpose of this? Isn't that only a different representation of the same EventType?